### ISSUE TITLE
NEW X Firemware without loader.sin

### ISSUE THREAD

**Issue by macallen** [Androxyde/devices#37]
I have got the new X Series firmware from emma(internal software of SM). But there is no loader.sin any more. So how to creat FTF? and Flash? Need HELP

**Comment by Bin4ry** [comment]
could you please upload all Emma files somewhere to analyse for us? just
the whole dB folder to be sure there is no new fileset. I don't have access
to internal firmwares and so I have not seen any x firmwares yet.

best regards

M.Allen <notifications@github.com> schrieb am Fr., 29. Apr. 2016, 17:38:

**Comment by macallen** [comment]
yes, of course!  I will upload files, so could u leave me a G+ or Hangout or any other else? then I will send u the download link, when I uploaded.

**Comment by Bin4ry** [comment]
could you send me the link via pm on Xda forums?
http://forum.xda-developers.com/member.php?u=1346722
but I am not able to work on it before next week.

regards

M.Allen <notifications@github.com> schrieb am Fr., 29. Apr. 2016, 18:01:

**Comment by macallen** [comment]
sure. 
And happy weekend.

**Comment by ty87998666** [comment]
Is loader.sin available now?I was trying to send private message on xda to u but u refused to accept users have less than 10 posts..v urgent,my phone is down,needs that loader,sin

**Comment by Bin4ry** [comment]
New x devices are supported for some time. Just download latest version of
flash tool and do it. No worries about loader 😉

Best

ty87998666 <notifications@github.com> schrieb am Do., 29. Sep. 2016, 09:39:

**Comment by ty87998666** [comment]
BUT THE FLASH TOOL KEPT TELLING ME THAT IT CAN NOT FIND THE LOADER.SIN

**Comment by Bin4ry** [comment]
Please upload your log.

ty87998666 <notifications@github.com> schrieb am Do., 29. Sep. 2016 um
13:22 Uhr:

**Comment by Androxyde** [comment]
what version of flashtool are you using ?

2016-09-29 13:22 GMT+02:00 ty87998666 <notifications@github.com>:

**Comment by dmarinon** [comment]
same problem here, I need to build a ftf file from a firmware folder downloaded with xperifirm, but flashtool procedure gets stuck on last part, when it needs to load loader.sin
X Series (Fxxx) folders in .flashTool/devices do not contain any loader.sin files!

**Comment by Androxyde** [comment]
What version of flashtool are you using?

**Comment by dmarinon** [comment]
I was using a previous version of flashtool.... After updating to the latest version, it works fine now. I found out new firmwares don't have nor need loader.sin files, but only latest version of flashtool knows that ;)

**Comment by Androxyde** [comment]
That's why I ask about which version you are using :)

**Comment by JuanCruzCDG** [comment]
Hi guys, I'm trying to flash a Sony Xperia XA1 with the stock ROM and although I read that Flashtool no longer requires the loader.sin it still throws the warning "No loader found or manually loaded" when I try to flash it. I'm using v0.9.33. Any tips?
